README.md

```diff
@@ -154,25 +152,30 @@ Here is a good editor for regexp, maybe try and do it without using it first tho
 ```ruby
 # /User/gollum/precious/ring.rb
 File.dirname(__FILE__)
+
+File.dirname("/User/gollum/precious/")
```

Comment:
So for this one `__FILE__` is a ruby keyword and returns the name of the file being executed `ring.rb`. This can also include the directory if you are running it like:
```
ruby gollum/precious/ring.rb
```
in which case `__FILE__` will be `gollum/precious/ring.rb`
https://ruby-doc.org/docs/keywords/1.9/Object.html#method-i-__FILE__

Reply:
Oh and `dirname` will give the directory name, so if `__FILE__` is just the filename only you will get `.`.